### PR TITLE
documentation-fix

### PR DESCRIPTION
**Pull request by gnawinganimal**
Fix documentation error

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -1,699 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SUPERCILEX) [inline .idea/codeStyleSettings.xml:1]: Oh lol, you deleted the code style. Can you try resetting your HEAD and making the doc fix again?

**Review by SUPERCILEX — CHANGES_REQUESTED** [review]

**Comment by andrewda** [comment]
We should keep the project settings, right?

**Comment by SUPERCILEX** [comment]
Yes, just the code style.

**Comment by gnawinganimal** [comment]
Ok, I made another one that DOESN'T mess with the styling. Lol, sorry about that.

**Comment by SUPERCILEX** [comment]
Well that was painful, congrats! 😄

**Pull request closed without merging**